frontend/src/components/TracingIntegration/TracesFetcher.ts

```diff
@@ -72,6 +72,15 @@ export class TracesFetcher {
             'Loading traces for all clusters. Tracing is not configured to store traces per cluster.'
           );
         }
+        const firstTraceTimestamp = Math.min(...traces.map(s => s.startTime));
+        // If the first trace starts more than one minute later than the start date that the user selects, show an info message
+        // The order in which traces are returned is not deterministic and
```

Comment:
Maybe a better test would be to compare the time period with the delta time of the returned traces?  And if coverage was bad then warn the user.  For example, let's say the requested time period is `Last 30m` and it's bad if the returned traces cover < 90% of the time period.  So, if `t(newestTrace)-t(oldestTrace) < 0.9*30` then show warning.

This basically is saying, if our returned traces cover enough the requested range, then good. It wouldn't matter if they were the oldest or the newest, just that they didn't all lump to one time slice.